gnosis/safe/api/transaction_service_api/transaction_service_api.py

```diff
@@ -204,13 +211,10 @@ def get_transactions(
 
         if safe_tx_hash_arg := kwargs.get("safe_tx_hash", None):
             safe_tx_hash = HexBytes(safe_tx_hash_arg)
-            if any(
-                self._build_transaction_service_tx(tx).safe_tx_hash != safe_tx_hash
+            [
```

Comment:
Add a comment explaining what are you doing here

Reply:
Yes, it's clearer. I totally agree. Added.